<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bit Operation Utility</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Hex Value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leHex"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pbHex">
         <property name="text">
          <string>Dump Set Bits</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Bit Positions</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leBits"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pbBits">
         <property name="text">
          <string>Generate Hex Value</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="pteOutput"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>leHex</sender>
   <signal>returnPressed()</signal>
   <receiver>pbHex</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leBits</sender>
   <signal>returnPressed()</signal>
   <receiver>pbBits</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
